packages/convex-helpers/server/filter.ts

```diff
@@ -84,7 +84,9 @@ class QueryWithFilter<T extends GenericTableInfo>
       if (uniqueResult === null) {
         uniqueResult = result;
       } else {
-        throw new Error("not unique");
+        throw new Error(
+          `unique() query returned more than one result. First pair of results: ${uniqueResult._id} and ${result._id}`
+        );
```

Comment:
nice! Can you do the same technique from https://github.com/get-convex/convex-backend/pull/59#pullrequestreview-2679927198 to make it terser?